src/pretix/base/metrics.py

```diff
@@ -0,0 +1,160 @@
+from django.conf import settings
+
+if settings.HAS_REDIS:
+    import django_redis
+
+REDIS_KEY_PREFIX = "pretix_metrics_"
+
+r = django_redis.get_redis_connection("redis")
```

Comment:
I normally use ``rc`` but I seldomly use it in a module-global namespace so far. Also, I don't feel mutable objects are constants, so I'd go for a simple ``redis`` here to be honest.